packages/python-sdk/tests/test_file_download.py

```diff
@@ -0,0 +1,27 @@
+import filecmp
+from os import path
+from e2b import Session
+
+async def test_download():
+    file_name = "video.webm"
+    local_dir = "tests/assets"
+
+    local_path = path.join(local_dir, file_name)
+
+    session = await Session.create("Nodejs")
+
+    # Upload the file first (it's uploaded to /home/user)
+    uploaded_file_path = session.upload_file(
+        file=open(local_path, "rb")
+    )
```

Comment:
This is recommended way how to open files in Python, this will handle closing the file
```suggestion
    with open(localpath, "rb") as f:
        uploaded_file_path = session.upload_file(file=f)
```